crm_claim/views/crm_claim_views.xml

```diff
@@ -13,48 +13,48 @@
                 <field name="activity_date_deadline" />
                 <field name="activity_summary" />
                 <field name="categ_id" string="Type" />
-                <field name="date_deadline" attrs="{'invisible': [(1, '=', 1)]}" />
-                <field name="date_closed" attrs="{'invisible': [(1, '=', 1)]}" />
-                <field name="company_id" groups="base.group_multi_company" />
+                <field attrs="{'invisible': [(1, '=', 1)]}" name="date_deadline" />
```

Comment:
Why we need to change this?

Reply:
And I don't think it works properly. It shows only header, without data, it does not make any sense.